src/aims_ui/page_controllers/b_multiple_matches/utils/multiple_match_api_utils.py

```diff
@@ -98,6 +98,26 @@ def job_data_by_current_user(include_old_jobs):
   return final_jobs
 
 
+def count_active_jobs():
+  """ Count the number of jobs in progress """
+  # Maximum jobs set by environment variable (Wrapper defaults to 7)
+
+  jobcount = 0
+
+  # Get the list of running jobs
+  r = job_api('/jobs?status=in-progress')
+
+  # [{'jobid': 1, 'userid': ...},
+  jobs_json = r.json().get('jobs', [])
+
+  for job in jobs_json:
+    jobcount = jobcount +1
+    # any other info to be gathered?
+
+  print(f'Bulk Jobs Running: {jobcount}')
```

Comment:
I think logging at an "info" level should he used here instead of print

Reply:
agreed